lib/cloud_controller/serializer.rb

```diff
@@ -7,7 +7,13 @@ def serializes_via_json(accessor_method_name)
         string = send("#{accessor_method_name}_without_serialization")
         return if string.blank?
 
-        MultiJson.load string
+        begin
+          MultiJson.load string
+        rescue MultiJson::ParseError
+          logger = Steno.logger('cc.serializer')
+          logger.error("Failed to deserialize #{guid}")
```

Comment:
I guess it's safe to assume that there is always a `guid` - but as these methods are defined for different models, it might be useful to also log the model type along with the guid.

Reply:
Got it.